loadgen/topgun.py

```diff
@@ -1,11 +1,10 @@
-import threading
```

Comment:
This shouldn't need to be moved. Python imports are traditionally sorted alphabetically.

Reply:
It was auto formatted and now imports are in alphabetical order.

Reply:
Weird, was I reading that diff backwards, or maybe I can't do alphabetical order in my head!? Sorry for the noise.